src/data/nat/basic.lean

```diff
@@ -262,6 +262,10 @@ begin
   simp [H]
 end
 
+@[simp] lemma lt_one_iff {n : ℕ} :
+  n < 1 ↔ n = 0 :=
+⟨λ h, le_antisymm (le_of_lt_succ h) n.zero_le, λ h, lt_succ_of_le h.le⟩
+
```

Comment:
```suggestion
lemma lt_succ_iff_le {n m : ℕ} :
  n < nat.succ m ↔ n ≤ m :=
⟨le_of_lt_succ, lt_succ_of_le⟩

@[simp] lemma lt_one_iff {n : ℕ} :
  n < 1 ↔ n = 0 :=
lt_succ_iff_le.trans le_zero_iff
```
`lt_one_iff` should probably appear shortly below `le_zero_iff`, further up the file.

I'm a little surprised I can't find `lt_succ_iff_le`

Reply:
I pushed this change myself

Reply:
I took the liberty of modifying your changes to use the correct lemma name :)